src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateField.tsx

```diff
@@ -0,0 +1,100 @@
+import { IconButton } from '@mui/material';
+import { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Svg } from 'shared/components';
+import { StyledFlexTopCenter } from 'shared/styles';
+
+import { PhrasalTemplateFieldProps } from './PhrasalTemplateField.types';
+import { RenderedField } from './PhrasalTemplateRenderField';
+
+export const PhrasalTemplateField = ({
+  canRemove = false,
+  name = '',
+  onRemove,
+  responseOptions = [],
+  type = 'sentence',
+  ...otherProps
+}: PhrasalTemplateFieldProps) => {
+  const [showExpandedMenu, setShowExpandedMenu] = useState(false);
+  const menuRef = useRef<HTMLDivElement>(null);
+  const { t } = useTranslation('app');
+
+  const handleFocusMenu = () => {
+    setShowExpandedMenu(true);
+  };
+
+  const handleMouseEnterMenu = () => {
+    setShowExpandedMenu(true);
+  };
+
+  const handleHoverLeaveMenu = () => {
+    if (document.activeElement) {
+      const children = menuRef.current?.childNodes;
+      const activeElIsChild = [...(children ?? [])].includes(document.activeElement);
+
+      if (activeElIsChild) {
+        return;
+      }
+    }
+
+    setShowExpandedMenu(false);
+  };
+
+  const handleBlurMenu: React.FocusEventHandler<HTMLDivElement> = (e) => {
+    if (e.relatedTarget) {
+      const children = menuRef.current?.childNodes;
+      const relatedTargetIsChild = [...(children ?? [])].includes(e.relatedTarget);
+
+      if (relatedTargetIsChild) {
+        return;
+      }
+    }
+
+    setShowExpandedMenu(false);
+  };
+
+  return (
+    <StyledFlexTopCenter sx={{ gap: 0.8, width: '100%' }}>
+      <RenderedField name={name} responseOptions={responseOptions} type={type} {...otherProps} />
```

Comment:
There's a layout issue with field error states:

![CleanShot 2024-08-01 at 13 29 52@2x](https://github.com/user-attachments/assets/56978e89-b6cf-45e3-abab-a5b1449d0947)

_For some context as to why this happens:_ Normally in the Admin Panel, fields are laid out so that they have enough of a gap below them to accommodate the optional helper error text without a vertical content shift (achieved using a negative bottom margin). However, in this case, the space between this field and the subsequent one is too small and causes the helper text to overlap with the field below.

This can be easily fixed here by undoing the helper text's negative bottom margin, and actually adding a few pixels to match the [design](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=650-60681&t=loPUvDkZwCxKa53T-4):

```suggestion
      <RenderedField
        name={name}
        responseOptions={responseOptions}
        type={type}
        sx={{ '.MuiFormHelperText-root': { mb: 0.4 } }}
        {...otherProps}
      />
```